articles/cosmos-db/change-feed-hl7-fhir-logic-apps.md

```diff
@@ -1,226 +1,226 @@
+	        )
+	        {
+	            var collectionLink = UriFactory.CreateDocumentCollectionUri(databaseId, collectionId);
+	
+	            var context = new DocumentDbContext();	
+	
+	            var docs = new List<dynamic>();
+	
+	            var partitionKeyRanges = new List<PartitionKeyRange>();
+	            FeedResponse<PartitionKeyRange> pkRangesResponse;
+	
+	            do
+	            {
+	                pkRangesResponse = await context.Client.ReadPartitionKeyRangeFeedAsync(collectionLink);
+	                partitionKeyRanges.AddRange(pkRangesResponse);
+	            } while (pkRangesResponse.ResponseContinuation != null);
+	
+	            foreach (var pkRange in partitionKeyRanges)
+	            {
+	                var changeFeedOptions = new ChangeFeedOptions
+	                {
+	                    StartFromBeginning = startfromBeginning,
+	                    RequestContinuation = null,
+	                    MaxItemCount = maximumItemCount,
+	                    PartitionKeyRangeId = pkRange.Id
+	                };
+	
+	                using (var query = context.Client.CreateDocumentChangeFeedQuery(collectionLink, changeFeedOptions))
+	                {
+	                    do
+	                    {
+	                        if (query != null)
+	                        {
+	                            var results = await query.ExecuteNextAsync<dynamic>().ConfigureAwait(false);
+	                            if (results.Count > 0)
+	                                docs.AddRange(results.Where(doc => doc.resourceType == resourceType));
+	                        }
+	                        else
+	                        {
+	                            throw new HttpResponseException(new HttpResponseMessage(HttpStatusCode.NotFound));
+	                        }
+	                    } while (query.HasMoreResults);
+	                }
+	            }
+	            if (docs.Count > 0)
+	                return docs;
+	            var msg = new StringContent("No documents found for " + resourceType + " Resource");
+	            var response = new HttpResponseMessage
+	            {
+	                StatusCode = HttpStatusCode.NotFound,
+	                Content = msg
+	            };
+	            return response;
+	        }
+	    }
+	}
+	
+```
+
+### Testing the FhirNotificationApi 
+
+The following image demonstrates how swagger was used to test the [FhirNotificationApi](#api-app-source).
+
+![The Swagger file used to test the API app](./media/change-feed-hl7-fhir-logic-apps/hl7-fhir-testing-app.png)
+
+
+### Azure portal dashboard
+
+The following image shows all of the Azure services for this solution running in the Azure portal.
+
+![The Azure portal showing all the services used in this HL7 FHIR tutorial](./media/change-feed-hl7-fhir-logic-apps/hl7-fhir-portal.png)
+
+
+## Summary
+
+- You have learned that Azure Cosmos DB has native support for notifications for new or modifed documents and how easy it is to use. 
```

Comment:
Looks like the merge resolution overwrote another fix. I'll try and PR this peice again

Reply:
I corrected that issue in the subsequent commit.

Reply:
Thanks! When I opened the PR, GitHub just showed the diff for the first push